integration-tests/mongodb-panache-kotlin/src/main/kotlin/io/quarkus/it/mongodb/panache/reactive/person/resources/ReactivePersonEntityResource.kt

```diff
@@ -42,7 +42,7 @@ class ReactivePersonEntityResource {
 
     @POST
     fun addPerson(person: ReactivePersonEntity): Uni<Response> {
-        return person.persist()
+        return person.persist<ReactivePersonEntity>()
                 .map { Response.created(URI.create("/persons/entity${person.id}")).build() }
```

Comment:
but in this case, e.g., a wildcard wouldn't work because you need that type info in the `map {}`